.github/workflows/build_and_test.yaml

```diff
@@ -17,34 +17,34 @@ env:
   ENVOY_GATEWAY_DEV_IMAGE: envoyproxy/gateway-dev
   ENVOY_GATEWAY_DEV_TAG: latest
 jobs:
-  build-and-test:
+  lint:
```

Comment:
making all run under a single runner/job is intentional, can you please revert this

Reply:
Yes sure!

Reply:
Done